### PR TITLE
Bump version of `@apollographql/graphql-playground-react` to v1.7.39.

### LINKED ISSUE #4937: @apollographql/graphql-playground-html generates incorrect links if you pass a falsey version
1.6.26 has a problem, that is
if you don't want a fixed version number and set version option to "", you will get a wrong url

```typescript
const apolloServer = new ApolloServer({
  schema,
  context: fastify,
  playground: {
    version: "",
    cdnUrl: "",
    settings: {
      "editor.theme": "light",
    },
  },
});

fastify.register(fastifyStatic, {
  root: path.join(__dirname, "../node_modules/@apollographql/graphql-playground-react/"),
  prefix: "/@apollographql/graphql-playground-react/"
});
```
will contribute to
```html
<link rel="stylesheet" href="/@apollographql/graphql-playground-reactbuild/static/css/index.css">
```

I found a workaround for the moment，you must set the version option and set route correctly 

```typescript
const apolloServer = new ApolloServer({
  schema,
  context: fastify,
  playground: {
    version: "1.6.26", // must set version
    cdnUrl: "",
    settings: {
      "editor.theme": "light",
    },
  },
});

fastify.register(fastifyStatic, {
  root: path.join(__dirname, "../node_modules/@apollographql/graphql-playground-react/"),
  prefix: "/@apollographql/graphql-playground-react@1.6.26/", // must use corresponding version
});
```

1.6.29 has fixed this bug

### PR DESCRIPTION
**Pull request by abernix**
This version brings a number of [Updates] from the upstream fork but also improves the support for `graphql@15` features like interfaces implementing interfaces, while maintaining support for the Apollo Query Planner pane and avoiding some upstream dependency bumps to Redux dependencies which resulted in the inability to "Save" settings on the Settings panel.

[Updates]: https://github.com/apollographql/graphql-playground/compare/b0e8c3a~...6cb0e11

Also Fixes #4937

### CODE REVIEW

File: packages/apollo-server-core/src/playground.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export {
 // This specifies the version of `graphql-playground-react` that will be served
```
Comment (glasser) [inline packages/apollo-server-core/src/playground.ts:10]: Maybe the comments here should include `@apollographql/`?

**Review by glasser — COMMENTED** [review]
@abernix Your "updates" link also includes v1.7.36 — why not bump to that one? (And also a change to constants.ts that isn't in 1.7.36 yet.)

**Comment by glasser** [comment]
Might be worth addressing https://github.com/apollographql/apollo-server/issues/4937 at the same time?

File: packages/apollo-server-core/src/playground.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export {
...
 // from `graphql-playground-html`.  It's passed to ``graphql-playground-html`'s
 // renderPlaygroundPage` via the integration packages' playground configuration.
```
Comment (abernix) [inline packages/apollo-server-core/src/playground.ts:12]:
```suggestion
// This specifies the React version of our fork of GraphQL Playground,
// `@apollographql/graphql-playground-react`.  It is related to, but not to
// be confused with, the `@apollographql/graphql-playground-html` package which
// is a dependency of Apollo Server's various integration `package.json`s files.
//
// The HTML (stub) file renders a `<script>` tag that loads the React (guts)
// from a CDN URL on jsdelivr.com, which allows serving of files from npm packages.
// 
// The version is passed to `@apollographql/graphql-playground-html`'s
// `renderPlaygroundPage` via the integration packages' `playground` config.
```

File: packages/apollo-server-core/src/playground.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ export {
...
-const playgroundVersion = '1.7.33';
+const playgroundVersion = '1.7.35';
```
Comment (abernix) [inline packages/apollo-server-core/src/playground.ts:13]:
```suggestion
const playgroundVersion = '1.7.38';
```

File: packages/apollo-server-core/src/playground.ts (hunk 1 above)
Comment (abernix) [reply packages/apollo-server-core/src/playground.ts:10]: https://github.com/apollographql/apollo-server/pull/4932#discussion_r581718804

**Comment by abernix** [comment]
The omission of `1.7.36` (and only bumping to `.35`) was unintentional on my part.  Good catch.

I've bumped this to `1.7.38`, which includes https://github.com/apollographql/graphql-playground/commit/9eea58fdad969328c9c1373ff188abeb9df96533 which should fix @silenaker's reported issue in #4937 and updated the link in my PR comment and the body of this PR.

**Comment by abernix** [comment]
Ok, while it was correct to at least bump this PR to use `v1.7.36` as @glasser noted above, I didn't need to make the `v1.7.38` release (also worth noting that I burned 🔥 `v1.7.37` because the release failed due to network problems.

What I _should_ have done (since the commit I pulled in was for the HTML package!) was to release `@apollograpqhl/graphql-playground-html@1.6.27` (note the `-html`).  So I've done that!

For completeness, however, that necessitated a `@apollographql/graphql-playground-react` to be published as `v1.7.39` due to its dependence on `@apollographql/graphql-playground-html`, but it's worth noting that the dependency doesn't seem necessary for Apollo Server's usage and is just to remove any mystery about published vs. unpublished code.

To further remove any mystery about _why_ this wouldn't be running on a `v1.7.39`, I have updated this PR accordingly with that version.  There should, shortly, be a Renovate (bot) PR coming in with the update to `@apollographql/graphql-playground-html` which brings it to `v1.6.27`, and again updated the PR description.

File: packages/apollo-server-core/src/playground.ts [hunk 4]
```diff
@@ -7,10 +7,17 @@ export {
...
+// `@apollographql/graphql-playground-react`.  It is related to, but not to
+// be confused with, the `@apollographql/graphql-playground-html` package which
+// is a dependency of Apollo Server's various integration `package.json`s files.
+//
+// The HTML (stub) file renders a `<script>` tag that loads the React (guts)
+// from a CDN URL on jsdelivr.com, which allows serving of files from npm packages.
+// 
+// The version is passed to `@apollographql/graphql-playground-html`'s
+// `renderPlaygroundPage` via the integration packages' `playground` config.
+const playgroundVersion = '1.7.38';
```
Comment (abernix) [inline packages/apollo-server-core/src/playground.ts:20]:
```suggestion
const playgroundVersion = '1.7.39';
```

**Review by glasser — APPROVED** [review]

**Comment by glasser** [comment]
(probably worth a changelog entry?)

**Comment by abernix** [comment]
@glasser Agree! `CHANGELOG.md` done! Thanks!

**Comment by glasser** [comment]
I've published a release with this to version `2.21.1-alpha.0`. My plan is to release 2.21.1 on Monday. If you want to test that this fix works before then, try running the prerelease yourself, and provide feedback on https://github.com/apollographql/apollo-server/pull/4990

**Pull request merged**